### PR TITLE
First stab a Docker version of the Atelier Website.

### PR DESCRIPTION
**Pull request by northern-tester**
Hey Sophie

Can you pull this branch and try and run the container please?

## Docker

To build in the root dir, run the following:

```
docker build --rm -f Dockerfile .
```

To run:

```
docker run --rm -d -p 5000:3000 atelierwebsite
```
*Where 5000 is the host port and 3000 is the exposed port in the container

Thanks :)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Copy files to install
+ADD ./bin/ /app/bin/
+ADD ./data/ /app/data/
+ADD ./public/ /app/public/
+ADD ./routes/ /app/routes/
+ADD ./views/ /app/views/
+ADD ./app.js /app
+ADD ./package.json /app
+RUN ls /app
```
Comment (redball3) [inline Dockerfile]: wondering what the reason for this is?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Pull base image.
+FROM ubuntu:14.04
+
+# Install Node.js
```
Comment (redball3) [inline Dockerfile]: you can probably just set the base image to use Node8 (alpine) or boron then you dont need to do the extra install steps for node8 which will save time on spinning up the container.

Comment (redball3) [reply Dockerfile]: https://hub.docker.com/_/node/ is a good place to see all the node images we can use. if you dont specify a distro in the image name i think its either ubuntu or debian.

File: Dockerfile (hunk 1 above)
Comment (northern-tester) [reply Dockerfile]: Bit of debug, removed

**Pull request merged**